### ISSUE TITLE
operator log is flushed

### ISSUE THREAD

**Issue by lilyhe123** [oracle/weblogic-kubernetes-operator#284]
I found that the operator log is fragile to be flushed with same error/warning messages. I encountered this in different scenarios and the case below is only one of them. Basically we need to inspect all operator code to avoid log the same error/warning message again and again.

If a WebLogic server failed to start, the operator log is flushed with a warning message below.

{"timestamp":"05-15-2018T03:44:49.679+0000","thread":40,"level":"WARNING","class":"oracle.kubernetes.operator.wlsconfig.WlsRetriever$WithHttpClientStep","method":"apply","timeInMillis":1526355889679,"message":"Failed to read health information from server admin-server due to exception: javax.ws.rs.ProcessingException: URI is not absolute","exception":"","code":"","headers":{},"body":""}

I assume the warning message is a consequence of the server starting failure. This made it harder to find the real reason why the server is failed to start.

**Comment by simon-meng-cn** [comment]
WlsRetriever$WithHttpClientStep.apply method is the only place log the WLS_HEALTH_READ_FAILED warning message. The possible scenario is this method be called multiple times.

**Comment by alai8** [comment]
Yes, the method is called multiple times, from periodic checking of weblogic server health state by the operator.
What other error/warning messages did you notice?

**Comment by simon-meng-cn** [comment]
Only one warning message is repeated too many times in the log. There are no other error/warning messages.
